src/board_controller/pieeg/inc/pieeg_board.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <string>
+#include <thread>
+
+#include "board.h"
+#include "board_controller.h"
+#include "math.h"
+#include "socket_server_tcp.h"
```

Comment:
why does it need socket?

Reply:
Removed the unnecessary socket_server_tcp.h inclusion.